contracts/feature/TokenBundle.sol

```diff
@@ -4,5 +4,47 @@ pragma solidity ^0.8.0;
 import "./interface/ITokenBundle.sol";
 
 abstract contract TokenBundle is ITokenBundle {
+    uint256 public nextTokenIdToMint;
```

Comment:
We don't want to have opinionated handling of token IDs in our sdk features. We leave the handling of tokenIds to the top level contract that will be inheriting this `TokenBundle` feature e.g. `Multiwrap` in this case.

The reason for the above:
- Folks have product level preferences of how token IDs should be handled. We shouldn't make the assumption for the end user that token Ids will be assigned serially.
- The `TokenBundle` contract should not have special / privileged status as _the_ base contract that's going to specify how token IDs will work.